### DISCUSSION TITLE
Auth Scope with Prisma

### DISCUSSION THREAD

**Discussion by haleksandre** [hayes/pothos#621] [Q&A] [answered]
The context typing is always `never` when using `t.withAuth(async () => ({ ...scopes }) | boolean`  or `t.withAuth({ [$all|$any]: { ...scopes } })` & `t.prismaField({ ...options })` but works perfectly fine with `t.withAuth({ ...scope })` & `t.prismaField({ ...options })`..

The auth & the resolver still work as it should but I''m not quite sure if it's a typing issue or I'm simply not using `t.withAuth` correctly?

**Comment by hayes** [comment, marked as answer]
Just published a new version that should fix this issue
